packages/p2p/src/components/modal-manager/modals/create-ad-modal/ad-component.scss

```
@@ -0,0 +1,28 @@
+.ad-component {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    border-radius: 1.6rem;
+    border: 0.1rem solid $color-grey-5;
+    padding: 2.4rem;
+    width: 18rem;
+    @media only screen and (max-width: 767px) {
```

is this for mobile screens? if so we could just use
```suggestion
    @include mobile{
```
and do why is the max width 767px?